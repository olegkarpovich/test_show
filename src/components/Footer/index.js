import React from "react";

function Footer() {
    return (
        <section id="footer-block">
            <article id="footer-description">
                <p> Test task.</p>
            </article>
        </section>
    );
}

export default Footer;
