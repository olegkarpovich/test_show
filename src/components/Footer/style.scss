#footer-block {
    width: 100%;
    background: #C0FFFF;
    color: #fff;
    bottom: 0;
}