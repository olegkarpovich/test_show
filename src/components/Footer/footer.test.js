import React from 'react';
import { render, act } from '@testing-library/react';
import Footer from './index';

const component = () => (
    <Footer />
);

describe('Footer component', () => {
    it('should render footer', () => {
        const { container } = render(component());

        expect(container).toMatchSnapshot();
    });
});