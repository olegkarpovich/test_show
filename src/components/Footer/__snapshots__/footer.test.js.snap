// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EpisodeItem component should render episode details page 1`] = `
<div>
  <section
    id="footer-block"
  >
    <article
      id="footer-description"
    >
      <p>
         Test task.
      </p>
    </article>
  </section>
</div>
`;

exports[`Footer component should render footer 1`] = `
<div>
  <section
    id="footer-block"
  >
    <article
      id="footer-description"
    >
      <p>
         Test task.
      </p>
    </article>
  </section>
</div>
`;
